benchmark/benchmark.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const Benchmark = require('benchmark');
+const benchmarks = require('beautify-benchmark');
+const nock = require('nock');
+
+Benchmark.options.maxTime = 0;
+Benchmark.options.minSamples = 5;
+
+nock.disableNetConnect();
```

Comment:
why using nock ?

Reply:
This was added to avoid any real `http` requests, but I think this is not required. Removed it.